sql/moz-fx-data-shared-prod/telemetry_derived/ech_adoption_rate_v1/query.sql

```diff
@@ -109,29 +110,21 @@ WITH unioned AS (
     submission_date,
     label,
     key,
-    value,
     metadata.geo.country
 )
 SELECT
   submission_date,
   metric,
   label,
   key,
-  value,
-  country_code,
-  SUM(client_count) AS total_client_count
+  SUM(handshakes) AS handshakes,
+  SUM(client_count) AS total_client_count,
+  country_code
 FROM
   unioned
 GROUP BY
```

Comment:
Not really relevant to this PR, but technically `GROUP BY ALL` is out of preview now so we can use it in prod queries if you want to make them a bit shorter:
https://cloud.google.com/bigquery/docs/reference/standard-sql/query-syntax#group_by_all

Reply:
`GROUP BY ALL` is still against our sql style guide of being explicit (although the guide can change).  IMO there isn't a benefit to making the ETL queries slightly shorter and it's better to have all the columns for readability and make changes to the query a little more intentional, i.e. make you think for a few extra seconds about what you're doing when making changes to ETL